src/Build/Definition/Project.cs

```diff
@@ -1417,6 +1417,11 @@ public void SaveLogicalProject(TextWriter writer)
             implementation.SaveLogicalProject(writer);
         }
 
+        public void PrintTargets(TextWriter writer)
```

Comment:
I'd rather have the Project API only return targets, and not deal with the extra problem domain of printing object model things to streams.